### ISSUE TITLE
Providing rules on the use of the new name

### ISSUE THREAD

**Issue by Zimmi48** [coq/rocq-prover.org#17]
Should we add some explicit rules about how to use the new name, and if so, where? In an upcoming renaming announcement? And in the meantime, in the renaming section of "About Rocq"?

The rules could be the following:

- The full name is the Rocq Prover (with this casing). Use it at least once in the introduction of any document or paper about Rocq.
- Never use the casing Rocq prover. Same for the Rocq platform. Use the Rocq Platform instead (or the Platform for short).
- If a sentence starts by Rocq + verb, prefer using the full name The Rocq Prover instead.
- Never say "Rocq 8". Past, Coq versions are still called Coq.

**Comment by SkySkimmer** [comment]
s/rules/gudelines/ unless we want to enforce them IMO

**Comment by Zimmi48** [comment]
Indeed, these guidelines won't be enforced, except in our own documentation.

**Comment by mattam82** [comment]
I guess we still need this in the website, it was not really touched upon in the changes file...

**Comment by mattam82** [comment]
@Zimmi48 wdyt?

**Comment by Zimmi48** [comment]
Indeed, it looks like the best place to add this would be https://rocq-prover.org/about#Name. We could also link to it from https://rocq-prover.org/changelog/2025-03-12-rocq-9.0. Alternatively, we could add a section about it in https://rocq-prover.org/logo, since it is already linked from https://rocq-prover.org/doc/V9.0.0/refman/changes.html#renaming-advice.
